### DISCUSSION TITLE
DevServices and Mail extension

### DISCUSSION THREAD

**Discussion by lordofthejars** [quarkusio/quarkus#37635] [Community]
I was thinking that would be great to have an integration between the Mail extension and DevServices, for example, starting the https://github.com/mailhog/MailHog container and exposing its UI, so we can send emails in Quarkus Dev mode too. It's just an idea, in the future, it could even implement a Dev UI like what happens with Kafka, to validate everything.  I could try to implement the DevService part, not sure about Dev UI :D

**Comment by gsmet** [comment]
See https://docs.quarkiverse.io/quarkus-mailpit/dev/index.html :)

**Comment by lordofthejars** [comment]
Oh great thanks; I searched but couldn't find it. I need it for running the Langchain4j email tool service example without having to install anything :)
